### DISCUSSION TITLE
blueretro buttons

### DISCUSSION THREAD

**Discussion by itslemcy** [darthcloud/BlueRetro#182] [Q&A]
hi @darthcloud , i haven't found in documentation references for the two buttons on your devkit pcb, i would like to use buttons to enter pairing mode, turn on or off the blueretro, reboot the esp32, etc.. is it possible? what are the functions and to what ios do they connect? thanks dude

**Comment by darthcloud** [comment]
Yes in wiki first page:
https://github.com/darthcloud/BlueRetro/wiki#esp32-buttons-usage

**Reply by darthcloud** [reply]
Boot is IO0. EN is EN

**Reply by itslemcy** [reply]
this is embarrassing, thanks a lot!
